src/ProxyClient/Varnish.php

```diff
@@ -151,6 +160,8 @@ protected function configureOptions()
         $resolver->setDefaults([
             'tags_header' => self::DEFAULT_HTTP_HEADER_CACHE_TAGS,
             'header_length' => 7500,
+            'tags_invalidation_by_purge' => false,
+            'tags_separator' => ',',
```

Comment:
is making this configurable related to the xkey change? what is the reason you want to do this, handle tags with `,` in them? i thought we remove the separator at some point from tags, to be sure.

if there is a strong argument for it, we probably should fix that bit before 2.0.